docs/manage/reference/accounts.md

```diff
@@ -0,0 +1,45 @@
+---
+linkTitle: "Account Management"
+title: "Account Management"
+weight: 1000
+layout: "docs"
+type: "docs"
+no_list: true
+description: "Log in and out of your Viam account."
+aliases:
+  - /fleet/account/
+  - /cloud/account/
+---
+
+The [Viam app](https://app.viam.com/) is a web UI for managing and building machines.
+
+## Create account and log In
+
+To get started on the Viam app, you must log in as an authorized user.
+Viam supports sign up using Google, GitHub, Apple, and email.
+
+Navigate to [the main page](https://app.viam.com/).
+If you haven't created an account yet, click **Sign Up** to create a new account using your preferred Single Sign On method or your email address and a password.
+If you already have an account, click **Log In** to log in using your Single Sign On credentials or your email address and password.
+
+If you forget your password to the app, click **Forgot password** and enter your email address to obtain instructions to reset your password.
+
+{{< alert title="Info" color="info" >}}
+Accounts created from separate authentication sources are unique to each other.
+{{< /alert >}}
+
+<br>
+
+## Sign out
+
+To log out or sign out of the [Viam app](https://app.viam.com/), click on your profile icon in the upper right corner of your browser window.
+Click **Sign out** to sign out of accessing all organizations, locations, and machines your credentials manage.
+
+## Delete account
+
+To delete your account, you can contact Viam Support at [contact@viam.com](mailto:contact@viam.com).
+
+Some important things to note:
+
+- Account termination may result in the destruction of any associated content
+- If you delete your robot, organization, or account by mistake, contact [contact@viam.com](mailto:contact@viam.com) immediately - they will try to help but cannot guarantee recovery or restoration.
```

Comment:
I think adding next steps with links to RBAC and to CLI authentication would be good

Reply:
We link to rbac at the top which is enough I think. For both that and the cli there's steps in between. Before rbac matters you'd need to create machines and organizations. And before using the cli you also very likely need to create a machine. 

I think any next steps after converting account deletion would be very strange.

Reply:
Yeah now that we link to rbac all good there. I was just thinking someone might come to this page and wonder about other kinds of logins, so I should've suggested more of a see also reference to CLI auth, not a next step in a sequential way. Fair enough.